base/applications/cmdutils/where/lang/it-IT.rc

```diff
@@ -0,0 +1,54 @@
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
```

Comment:
```suggestion
/* 
 * PROJECT:     ReactOS WHERE command
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Italian resource file
 * TRANSLATOR:  Copyright 2023 Carlo Bramini
 */

LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
```
Also, usually, we place an email address after the "copyright" notice. Do you want to put one?